cardano-constitution/data/defaultConstitution.schema.json

```diff
@@ -2,17 +2,25 @@
     "$schema": "https://json-schema.org/draft/2019-09/schema",
     "type": "object",
     "propertyNames": {
-        "pattern": "^[0-9]+$"
+        "$comment": "The param id is supposed to be a non-negative integer,"
+        , "$comment": "but json only allow strings key names (not integers)."
+        , "$comment": "We enforce with a regex (^(0|[1-9][0-9]*)$) the param id to the non-negative decimal numbers"
+        , "$comment": "The regex also disallows leading zeroes, to avoid silent duplicates"
+        , "pattern": "^(0|[1-9][0-9]*)$"
     },
     "additionalProperties": {
         "type": "object",
         "properties": {
-            "____comment____": { "type": "string" },
+            "$comment": { "type": "string" },
             "predicates": { "type": "object"
                             , "properties": {
                                 "minValue": { "type": "integer" },
```

Comment:
The limited semantic will become an issue for bounds that should be valid, to those that should not.